workflow/rules/common.smk

```diff
@@ -212,7 +213,15 @@ def get_fastqs(wildcards):
 
 
 def get_resource(name):
-    return str((Path(workflow.snakefile).parent.parent.parent / "resources") / name)
+    if "https" in str(Path(workflow.snakefile)):
+        dirname = os.path.dirname
+        path_snakefile = str(workflow.snakefile)
+        path_new = dirname(dirname(path_snakefile))
+        folder = "resources"
+        path_complete = f"{path_new}/{folder}/{name}"
+        return path_complete
+    else:
+        return str((Path(workflow.snakefile).parent.parent.parent / "resources") / name)
```

Comment:
This is much simpler, and also more portable as it also works in the cloud:
```suggestion
    return workflow.source_path(f"../../resources/{name}")
```
In addition, I suggest to move the resources subfolder into the workflow subfolder in the future. This makes the separation between resources that come together with the workflow and resources that are downloaded by the workflow cleaner.